dice-game-ui-angular/src/app/services/game-storage-service.ts

```diff
@@ -2,10 +2,25 @@ import { Injectable } from '@angular/core';
 import { Guid } from 'guid-typescript';
 import { GameEngine } from 'engine';
 
+export class SavedGameEngine {
+  constructor(gameEngine: GameEngine) {
+    this.gameEngine = gameEngine;
+    this.guid = Guid.create().toString();
+    this.creationDate = new Date();
+  }
+
+  gameEngine: GameEngine;
```

Comment:
Fields should be placed before a constructor.

Reply:
Changed